### ISSUE TITLE
syntax for evaluation mode on custom data?

### ISSUE THREAD

**Issue by gbrow004** [pytorch/examples#586]
I fine-tuned a model on a custom dataset that was pretrained with Imagenet. Now that I have the model and resulting best epoch in a pth file. What is the correct syntax for evaluation on the validation dataset using my new model. I know I need to set the '-e' flag, butI don't see how to point the script to my newly-trained model. Apologies if this is a novice issue.

**Comment by msaroufim** [comment]
You can take a look at this https://github.com/pytorch/examples/blob/main/imagenet/main.py#L236

you're basically pointing -e to a directory from which the script will create a validation data loader here https://github.com/pytorch/examples/blob/main/imagenet/main.py#L226
